### PR TITLE
Update Profiler Images and Videos to Reflect New Nav

### PR DESCRIPTION
**Pull request by brett0000FF**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: static/images/profiler/code_hotspots_tab-timeline.png
Comment (gturbat) [inline static/images/profiler/code_hotspots_tab-timeline.png:1]: Let's use a /GET train example, more interesting to see

File: static/images/profiler/endpoint_agg.png
Comment (gturbat) [inline static/images/profiler/endpoint_agg.png:1]: - Can we use the CPU profile type instead?
- I think we should hover the endpoint name on the right, to highlight that we filtered on it
- We barely see the FG in this image, is there a vertical cropping or this is the full size of the screen?

File: static/images/profiler/endpoint_per_request.png
Comment (gturbat) [inline static/images/profiler/endpoint_per_request.png:1]: I think we need to keep the video here, otherwise it is not clear how you end up here

**Comment by brett0000FF** [comment]
@gturbat - Good suggestions. I updated the images for `static/images/profiler/code_hotspots_tab-timeline.png` and `static/images/profiler/endpoint_agg.png`. Please let me know how this looks now!

Also, I was struggling to find an interesting example to make the video for `static/images/profiler/endpoint_per_request.png`. Could you please help me grab a video for this one, without recording the left-nav ideally? Thanks!

**Comment by gturbat** [comment]
> @gturbat - Good suggestions. I updated the images for `static/images/profiler/code_hotspots_tab-timeline.png` and `static/images/profiler/endpoint_agg.png`. Please let me know how this looks now!
> 
> Also, I was struggling to find an interesting example to make the video for `static/images/profiler/endpoint_per_request.png`. Could you please help me grab a video for this one, without recording the left-nav ideally? Thanks!

@brett0000FF wdyt of this?
https://github.com/DataDog/documentation/assets/45032269/f66503b9-0e81-4b11-ae9d-ae17e2b82f94

**Comment by brett0000FF** [comment]
> @brett0000FF wdyt of this? https://github.com/DataDog/documentation/assets/45032269/f66503b9-0e81-4b11-ae9d-ae17e2b82f94

@gturbat -- Oo, nice--that looks great. I can pop this new video back in place. Thanks!

**Review by rtrieu — APPROVED** [review]
Looks good!

**Pull request merged**